jwt_verify_lib/status.h

```diff
@@ -219,6 +219,13 @@ enum class Status {
  */
 std::string getStatusString(Status status);
 
+/**
+ * Convert enum status to string name.
+ * @param status is the enum status.
+ * @return the status name.
+ */
+std::string getStatusName(Status status);
```

Comment:
Why is this needed?  Could you describe use case?

Reply:
Follow the convention of response code details which is one concise word in snake case.

Reply:
Try to use getStatusString().  If not enough, we can improve it.

Reply:
The enum name doesn't mean much.  Its string should carry detail info.  Can we just put the StatusString into rc_detail.

Reply:
Agree it isn't as informative but almost all the filters use this kind of condensed word as reponse_code_detail.

I think it is doable . Lemme have a PR and see if upstream envoy accept it.